multimodal/setup.py

```diff
@@ -55,23 +55,31 @@
     "jinja2>=3.0.3,<3.2",
     "tensorboard>=2.9,<3",
     "pytesseract>=0.3.9,<0.3.11",
-    "PyMuPDF<=1.21.1",
 ]
 
 install_requires = ag.get_dependency_version_ranges(install_requires)
 
+tests_require = [
+    "black>=22.3,<23.0",
+    "isort>=5.10",
+    "datasets>=2.3.2,<=2.3.2",
+    "onnx>=1.13.0,<1.14.0",
+    "onnxruntime>=1.13.0,<1.14.0;platform_system=='Darwin'",
+    "onnxruntime-gpu>=1.13.0,<1.14.0;platform_system!='Darwin'",
+    "tensorrt>=8.5.3.1,<8.5.4;platform_system=='Linux'",
```

Comment:
`onnx` and `tensorrt` are not only for testing.

Reply:
They were originally in `tests`, I have not modified the logic from the original. You may consider editing this logic in a follow-up PR, as the purpose of this PR is specifically to move PyMuPDF to optional.